store/mockstore/mocktikv/mock_tikv_test.go

```diff
@@ -33,24 +34,81 @@ type testMockTiKVSuite struct {
 
 type testMarshal struct{}
 
+// testMvccStore is used to test MvccStore implementation.
+type testMvccStore struct {
+	testMockTiKVSuite
+}
+
+// testMVCCLevelDB is used to test MVCCLevelDB implementation.
+type testMVCCLevelDB struct {
+	testMockTiKVSuite
+}
```

Comment:
Friendly remind: avoid to move code if it's not necessary.
Because it would make the code reviewer's work easier.